src/test/java/TSibatyykEvhenii/Pages/HotlineSite.java

```diff
@@ -0,0 +1,41 @@
+package TSibatyykEvhenii.Pages;
+
+import BasePageAndTestInit.BasePage;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.util.List;
+
+public class HotlineSite extends BasePage {
```

Comment:
Will u write all elements and methods in one class? 
change the name, because it ruins Page Object Pattern